contracts/cw20-bonding/src/curves.rs

```diff
@@ -218,14 +218,14 @@ impl DecimalPlaces {
         }
     }
 
-    pub fn to_reserve(&self, reserve: Decimal) -> Uint128 {
+    pub fn to_reserve(self, reserve: Decimal) -> Uint128 {
```

Comment:
I'll let @orkunkl defend this one :D

Reply:
Just checked and clippy doesn't complain about `&self`.

I guess I didn't question the change because the type is `Copy`, feels fairly primitive and is small enough that (on 64-bit machines) there's no gain passing it by reference. Feels right somehow. But probably doesn't really matter.

https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=c0dd7e30065ca2135e8bce172cdda8c0

Reply:
Yes it was requested by clippy and applied it.

Reply:
local clippy version: `0.1.53 (53cb7b09 2021-06-17)`

Reply:
Oh, good catch. `0.1.51` doesn't complain, but `0.1.53` does.